specification/kubernetesconfiguration/resource-manager/Microsoft.KubernetesConfiguration/preview/2022-01-15-preview/extensionTypes.json

```diff
@@ -0,0 +1,379 @@
+          "LocationExtensionTypes_List_MaximumSet_Gen": {
+            "$ref": "./examples/LocationExtensionTypes_List.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ExtensionLocationParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v2/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ExtensionTypeList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ExtensionTypeList": {
+      "description": "List Extension Types",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "The list of Extension Types",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ExtensionType"
+          }
+        },
+        "nextLink": {
+          "description": "The link to fetch the next page of Extension Types",
+          "type": "string"
+        }
+      }
+    },
+    "ExtensionType": {
+      "type": "object",
+      "description": "Represents an Extension Type.",
+      "required": [
+        "properties"
+      ],
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ExtensionTypeProperties",
+          "description": "Describes the Resource Type properties."
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/systemData",
+          "readOnly": true,
+          "description": "Metadata pertaining to creation and last modification of the resource."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "ExtensionTypeProperties": {
+      "description": "Properties of the connected cluster.",
+      "type": "object",
+      "properties": {
+        "releaseTrains": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Extension release train: preview or stable"
+        },
+        "clusterTypes": {
+          "description": "Cluster types",
+          "readOnly": true,
+          "type": "array",
```

Comment:
>array [](http://example.com/codeflow?start=19&length=5)

This is a cross-version breaking change. You will need to get a breaking change review.

Reply:
So this service was never released to production before this point. The ARM Manifest has still not been updated with these new resource types. Is breaking change review still required?